pom.xml

```diff
@@ -23,7 +23,7 @@
   </scm>
 
   <properties>
-    <springVersion>3.2.14.RELEASE</springVersion>
+    <springVersion>5.1.9.RELEASE</springVersion>
```

Comment:
I think it's probably better to remove the coupling with Spring in the tests so we can remove this dependency entirely. This will break one test (`BeforeSpringClassDataFolderIntegrationTest`) which I think we can live without. If you agree @patduin I'll close this PR and raise a separate one to remove the Spring dependencies.

Reply:
yes agree